cli/bazelisk/bazelisk.go

```diff
@@ -180,6 +181,30 @@ func makePipeWriter(w io.Writer) (pw *os.File, closeFunc func(), err error) {
 	return
 }
 
+// Redirects either os.Stdout or os.Stderr to the given writer. Calling the
+// returned close function stops redirection.
+func redirectStdio(w io.Writer, stdio **os.File) (close func(), err error) {
```

Comment:
This func needs to assign to `os.Stdout`, so it takes a pointer argument to `os.Stdout` so that it can do `*stdio = ...` to assign to it. But `os.Stdout` itself has type `*os.File`, so it needs to take a `**os.File`.